src/components/shell/NavigationMenu.tsx

```diff
@@ -0,0 +1,79 @@
+import { AccountCircle as AccountCircleIcon } from '@mui/icons-material'
+import { Box, Divider, IconButton, Typography } from '@mui/material'
+import Menu from '@mui/material/Menu'
+import MenuItem from '@mui/material/MenuItem'
+import * as React from 'react'
+import { useAuth } from '../../hooks/useAuth'
+
+interface NavigationMenuProp {}
+
+const NavigationMenu: React.FC<NavigationMenuProp> = () => {
+  const { user } = useAuth()
+
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null)
```

Comment:
Shouldn't this be a useRef() instead ?

Reply:
I'm not sure why we store the html element in a state ? Could we achieve this another way ?

Reply:
We are following MUI documentation on using the Menu component 
https://mui.com/material-ui/react-menu/